packages/fx-core/src/plugins/resource/apim/constants.ts

```diff
@@ -8,7 +8,7 @@ export class ProjectConstants {
   public static readonly pluginShortName: string = "APIM";
   public static readonly pluginName: string = "fx-resource-apim";
   public static readonly pluginDisplayName: string = "API Management";
-  public static readonly configFilePath: string = "env.default.json";
+  public static readonly configFilePath: string = "state.xxx.json";
```

Comment:
what's the purpose of this variable? the value seems strange... should we add env name to it dynamically?

Reply:
This config file path variable is used in error message of `EmptyConfigValue` error type. 

> Project configuration '${name}' of '${component}' is missing in **'${ProjectConstants.configFilePath}'**. Retry ${retryOperation} or set the value manually.


Since the original `env.default.json` does not exist, it is srtange to say somehing is missing in env.default.json. So update it to `state.xxx.json`. Any better idea?

Reply:
Revert the bug-fix and open a [bug](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12493238) for @XiaofuHuang .